clients/src/main/java/org/apache/kafka/clients/admin/ConfigEntry.java

```diff
@@ -167,6 +169,8 @@ public int hashCode() {
         result = prime * result + (isReadOnly ? 1 : 0);
         result = prime * result + source.hashCode();
         result = prime * result + synonyms.hashCode();
+        result = prime * result + type.hashCode();
+        result = prime * result + documentation.hashCode();
```

Comment:
documentation field could be null so we must handle it here.

Reply:
nice point! will copy that!